src/metpy/calc/thermo.py

```diff
@@ -449,9 +449,9 @@ def ccl(pressure, temperature, dewpoint, height=None, mixed_layer_depth=None, wh
     r"""Calculate the convective condensation level (CCL).
```

Comment:
One last thing (sorry), can we make this explicit to help discovery:
```suggestion
    r"""Calculate the convective condensation level (CCL) and convective temperature.
```